### DISCUSSION TITLE
TSML 3.14.6

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#920] [Q&A] [answered]
Seems this update adds **Location Temporarily Closed** on the list and detail pages to all meetings that are online and still have a street address. Was this done to encourage admins to clean up their meetings considering that 2-1/2 years into this that the meeting may never return to the location it once met at? 

We discussed this on the TIAA forum. See https://discourse.tiaa-forum.org/t/meeting-guide-app-update-summer-2022/7821/3

**Comment by joshreisner** [comment, marked as answer]
no, this is actually a side effect of [your request](https://github.com/code4recovery/12-step-meeting-list/discussions/904) to have local "flag" settings propagate to TSML UI

by default, TSML has long had "Location Temporarily Closed" as one of its flagged types, but this was not propagating before to TSML UI.

at any rate, that code snippet should now work for you

**Reply by Grateful-Viv** [reply]
Thanks for the clarification, @joshreisner
